packages/circuit-ui/components/Button/Button.spec.tsx

```diff
@@ -83,6 +83,15 @@ describe('Button', () => {
     });
   });
 
+  it('should render with loading styles', () => {
+    const wrapper = renderButton(create, {
+      ...baseProps,
+      isLoading: true,
+      loadingLabel: 'Loading',
+    });
+    expect(wrapper).toMatchSnapshot();
```

Comment:
In addition to the snapshot, I'd also add other checks here e.g. to verify that the loading label is rendered, that it's accessible, etc.